bears/python/requirements/PySafetyBear.py

```diff
@@ -40,15 +55,33 @@ class PySafetyBear(LocalBear):
         'Python 3 Requirements',
     }
     AUTHORS = {'Bence Nagy'}
-    REQUIREMENTS = {PipRequirement('safety', '0.5.1')}
+    REQUIREMENTS = {PipRequirement('safety', '1.8.2')}
     AUTHORS_EMAILS = {'bence@underyx.me'}
     LICENSE = 'AGPL'
     CAN_DETECT = {'Security'}
 
-    def run(self, filename, file):
+    def setup_dependencies(self):
+        insecure_full_json_url = ('https://raw.githubusercontent.com/'
```

Comment:
put these in module level global _private variables so that people can access them easily if they need to.